docs/bokeh/Makefile

```diff
@@ -26,5 +26,7 @@ html:
 	&& echo \
 	&& echo "Build finished in $$(($$(date +%s)-start)) seconds. The HTML pages are in $(BUILDDIR)/html."
 
+test-accessibility:
```

Comment:
AFAIK `make` targets don't usually auto-complete in shells, so there's typically a preference for shorter target names. Suggest renaming this do just `audit`